client/src/containers/Course/CourseDetail.js

```diff
@@ -147,5 +144,5 @@ const mapStateToProps = (state) => ({
 
 export default connect(
   mapStateToProps,
-  { getProfRating, getCourseDescription }
+  { getBatchProfRating, getCourseDescription }
```

Comment:
move this to mapDispatchToProps for better readability, and adding new functions in the future